### ISSUE TITLE
double space not reported

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#662]
guess its due to keywords like TYPE + LIKE which can have preceding whitespace

```abap
SELECT SINGLE sdfds FROM but000 INTO sdfsd  WHERE partner = w_adr-partner.
```

**Comment by larshp** [comment]
also

```abap
read_view( IMPORTING es_dd25v        = ls_dd25v ).
```

**Comment by larshp** [comment]
hm, closing
